pkg/config/config.go

```diff
@@ -15,16 +15,17 @@ type Server struct {
 }
 
 type Config struct {
-	ApiServer                Server        `yaml:"apiServer"`
-	LogLevel                 string        `yaml:"logLevel"`
-	EnabledPlugin            []string      `yaml:"enabledPlugin"`
-	MetricsInterval          time.Duration `yaml:"metricsInterval"`
-	EnableTelemetry          bool          `yaml:"enableTelemetry"`
-	EnableRetinaEndpoint     bool          `yaml:"enableRetinaEndpoint"`
-	EnablePodLevel           bool          `yaml:"enablePodLevel"`
-	RemoteContext            bool          `yaml:"remoteContext"`
-	EnableAnnotations        bool          `yaml:"enableAnnotations"`
-	BypassLookupIPOfInterest bool          `yaml:"bypassLookupIPOfInterest"`
+	APIServer                          Server        `yaml:"apiServer"`
+	LogLevel                           string        `yaml:"logLevel"`
+	EnabledPlugin                      []string      `yaml:"enabledPlugin"`
+	MetricsInterval                    time.Duration `yaml:"metricsInterval"`
+	EnableTelemetry                    bool          `yaml:"enableTelemetry"`
+	EnableRetinaEndpoint               bool          `yaml:"enableRetinaEndpoint"`
+	EnablePodLevel                     bool          `yaml:"enablePodLevel"`
+	RemoteContext                      bool          `yaml:"remoteContext"`
+	EnableAnnotations                  bool          `yaml:"enableAnnotations"`
+	BypassLookupIPOfInterest           bool          `yaml:"bypassLookupIPOfInterest"`
+	AttachBPFProgramToDefaultInterface bool          `yaml:"attachBPFProgramToDefaultInterface"`
```

Comment:
I agree, @anubhabMajumdar, that it will be less confusing ("what BPF program?"). The question is what should the experience be? Should it be an enum like: "aggregationLevel: full" and "aggregationLevel: pod" or something? What's your vision?

Reply:
The current name is looking good - aggregationDataLevel.